### ISSUE TITLE
Pressure Level Embeddings

### ISSUE THREAD

**Issue by mc4117** [ecmwf/anemoi-training#54]
### Is your feature request related to a problem? Please describe.

The current code does not take into account embeddings in the z-dimension (either through height or pressure levels). We saw with Sara's work that pressure level weightings can have a large impact on results, especially in the upper atmosphere. 

This has been done in Aurora (https://arxiv.org/abs/2405.13063) which shows significant improvements to graphcast in the upper atmosphere, and also in https://arxiv.org/abs/2405.06004v1 and https://arxiv.org/pdf/2405.14527. 

### Describe the solution you'd like

Include pressure level embeddings in our encoder implementation and evaluate the impact of this.

### Organisation

ECMWF

**Comment by ssmmnn11** [comment]
Do we have the metadata accessible? Then it should be very straight forward to implement.

**Comment by mchantry** [comment]
Yes, we now have this meta-data with the latest version of anemoi datasets.

**Comment by mc4117** [comment]
Two paper that came out today that could be relevant when thinking about the vertical dimension 
https://gmd.copernicus.org/articles/17/8873/2024/
https://arxiv.org/pdf/2412.12971

**Comment by anaprietonem** [comment]
[heart]         Ana Prieto Nemesio reacted to your message:
________________________________
